frontend/src/routes/Upload.tsx

```diff
@@ -34,39 +45,50 @@ export default class upload extends Component {
   };
 
   isReady = () => {
-    return this.state.file !== "" && this.state.description !== "";
+    return (
+      this.state.file &&
+      this.state.description &&
+      this.state.legalShare &&
+      this.state.userData &&
+      this.state.openSource
+    );
   };
 
   onSubmit = () => {
-    if (this.isReady()) {
+    if (this.state.file && this.state.description) {
```

Comment:
Should be the same condition as in `isReady()`

Reply:
Fixed